README.md

```diff
@@ -1,25 +1,25 @@
 # karma-clone
 
-The aim of this exercise is to create a page that looks just like this one: 
+The aim of this exercise is to create a page that looks just like this one:
 
 ![Karma desktop design](./homepage-desktop.png)
 
 ### Set up
 
- - **Fork and clone** this repo.
```

Comment:
Usually you want have in the PR only changes related to the feature. Linter often change existing code, this should be avoided as it will make the file history harder to read (continuous changes on the "blame" section and history that are not feature related and wrong authors compared to the original author). This is , of course, more related to a work environment than this solo project, but thought to mention for the future :)